### ISSUE TITLE
Adding Suffix to Day e.g 5th 

### ISSUE THREAD

**Issue by thayeylolu** [mccutchen/strftime.org#30]
#29 
Hello @mccutchen . Thank you for sharing the link, but it does not contain what I am looking for. I meant to ask if strftime allows formatting a date with a suffix. Formatting examples like: May 5th, Jan 1st . I know it allows formatting like May 5, and Jan 1

**Comment by mccutchen** [comment]
Hello again!  If the [Python strftime docs](https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes) do not contain what you are looking for, then what you are looking for is not part of the Python standard library's `strftime` functionality and you will need to implement it yourself or find a 3rd party package that implements it for you.

In general, opening issues on this repo is not the best place to find the answers you're looking for.  Please check out the resources under **Got a Python problem or question?** on the official Python language help documentation:
https://www.python.org/about/help/#got-a-python-problem-or-question

I tend to use a search engine + Stack Overflow to answer questions like this.  For example, a [search for "python add ordinal suffix to datetime"](https://duckduckgo.com/?q=python+add+ordinal+suffix+to+datetime) yields some interesting results, including these Stack Overflow discussions of the exact question you have:
- https://stackoverflow.com/questions/5891555/display-the-date-like-may-5th-using-pythons-strftime
- https://stackoverflow.com/questions/3644417/python-format-datetime-with-st-nd-rd-th-english-ordinal-suffix-like

In short:  you can't do this without writing some custom code.
